recipes/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Run this from the root of the OpenTurbine repository e.g.
+# docker build -f recipes/Dockerfile -t openturbine:latest .
+
+FROM ubuntu:latest
+
+# Use bash as the default shell
+SHELL ["/bin/bash", "-c"]
+
+# Step 1: Install required packages and tools
+RUN apt-get update -qq && \
+    apt-get install -y \
+        build-essential \
+        clang-tidy \
+        cmake \
+        curl \
+        g++ \
+        gfortran \
+        git \
+        libcurl4-openssl-dev \
+        make \
+        python3 \
+        python3-pip \
+        unzip \
+        wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Do we have permissions to do this in Docker?  These commands typically would require `sudo`.

Reply:
I don't think you need `sudo` for this.